Core/Types/GameComparator/StrictGameComparator.cs

```diff
@@ -39,15 +34,11 @@ public override bool SingleVersionsCompatible(KspVersion gameVersion, CkanModule
                 }
                 else if (module.ksp_version_min != null)
                 {
-                    var minRange = module.ksp_version_min.ToVersionRange();
-
-                    moduleRange = new KspVersionRange(minRange.Lower, KspVersionBound.Unbounded);
+                    moduleRange = new KspVersionRange(module.ksp_version_min, new KspVersion());
```

Comment:
Could we re-use the static instance `KspVersion.Any` here instead of making a new one?
(same question for the change below)